cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -108,3 +108,11 @@ keystore_pass: changeme
 # like CentOS or RHEL <= 7, this is the yum conf file, but for Fedora and the
 # like, this is the dnf conf file
 version_pin_file: /etc/yum.conf
+# This is the port Jenkins will listen on for slaves.
+jenkins_slave_agent_port: 50000
+# This is a list of ports to allow in firewalld.
```

Comment:
It looks like we're actually using iptables for this instead of firewalld, so we may want to update this comment to reflect that.